setup.py

```diff
@@ -9,6 +11,15 @@
 with open('README.rst', 'r') as f:
     readme = f.read()
 
+if os.system('curl --version | grep -i nss') == 0:
```

Comment:
According to [1] the usage of `os.system` is not encouraged, subprocess module's functions should be used instead. That said, [2] can be used to grab the results and see if should be used NSS or OpenSSL for the pycurl installation.

Something like below should do the job:

```python
output = str(subprocess.check_output(('curl', '--version')))
if 'NSS' in output:
    curl_lib = 'nss'
elif 'OpenSSL' in output:
    curl_lib = 'openssl'
else:
    # ...
```

The above code may raise an exception if `curl` is not installed.

[1] https://docs.python.org/2/library/os.html#os.system

[2] https://docs.python.org/2/library/subprocess.html#subprocess.check_output